sdk/spring/azure-spring-boot-test-aad-obo/src/test/java/com/azure/test/aad/selenium/AADWebAppAndWebApiInOneAppIT.java

```diff
@@ -0,0 +1,176 @@
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.ImportAutoConfiguration;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.annotation.Order;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClient;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClientManager;
+import org.springframework.security.oauth2.client.annotation.RegisteredOAuth2AuthorizedClient;
+import org.springframework.security.oauth2.client.authentication.OAuth2AuthenticationToken;
+import org.springframework.security.oauth2.client.web.reactive.function.client.ServletOAuth2AuthorizedClientExchangeFilterFunction;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import java.security.Principal;
+import java.util.Collections;
+import java.util.Map;
+
+import static com.azure.spring.test.Constant.MULTI_TENANT_SCOPE_GRAPH_READ;
+import static com.azure.spring.test.EnvironmentVariable.AAD_MULTI_TENANT_CLIENT_ID;
+import static com.azure.spring.test.EnvironmentVariable.AAD_MULTI_TENANT_CLIENT_SECRET;
+import static com.azure.spring.test.EnvironmentVariable.AAD_USER_NAME_2;
+import static com.azure.spring.test.EnvironmentVariable.AAD_USER_PASSWORD_2;
+import static com.azure.test.aad.selenium.AADSeleniumITHelper.createDefaultProperties;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.springframework.security.oauth2.client.web.reactive.function.client.ServletOAuth2AuthorizedClientExchangeFilterFunction.oauth2AuthorizedClient;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class AADWebAppAndWebApiInOneAppIT {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AADWebAppAndWebApiInOneAppIT.class);
+    private static final String GRAPH_ME_ENDPOINT = "https://graph.microsoft.com/v1.0/me";
+
+    private Map<String, String> properties;
+
+    @BeforeAll
+    public void beforeAll() {
+        properties = createDefaultProperties();
```

Comment:
To make things simple, don't create abstract class. Just create `createProperties()` in `AADWebAppAndWebApiInOneAppIT`.

Reply:
OK